### PR TITLE
[APPTS-9598] remove grunt tasks and add eslint standard

### PR DESCRIPTION
**Pull request by jareddlc**
This PR will remove the grunt tasks and implement axway eslint standards

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Appcelerator Platform SDK for node.js",
   "main": "index.js",
   "scripts": {
-    "test": "NODE_ENV=development nyc -r text-summary --temp-directory ./coverage --clean mocha -R spec -t 40000 --check-leaks",
+    "test": "eslint --ext .js ./; NODE_ENV=development nyc -r text-summary --temp-directory ./coverage --clean mocha -R spec -t 40000 --check-leaks",
```
Comment (pkclark) [inline package.json]: @jareddlc why?

Comment (jareddlc) [reply package.json]: easiest way to get functionality without having to edit jenkins job. I could potentially add a Jenkinsfile that will only run the lint job

Comment (pkclark) [reply package.json]: @jareddlc change the Jenkinsfile instead.  The scripts should stay granular.

**Comment by pkclark** [comment]
@jareddlc why are you adding lock file to the repo?

**Comment by jareddlc** [comment]
jenkins uses npm@5.x.x and was causing issues without the lockfile since the first npm i will generate a lock file, and subsequest npm i will use the lock file generated, which causes issues

**Pull request closed without merging**